app/components/Request/Request.js

```diff
@@ -130,11 +140,12 @@ class Request extends React.Component {
     const { intl } = this.props
     return (
       <Box>
-        <Box pb={2}>
+        <Flex pb={2}>
           <Label htmlFor="memo">
             <FormattedMessage {...messages.memo} />
           </Label>
-        </Box>
+          <TooltipButton>{intl.formatMessage({ ...messages.memo_help })}</TooltipButton>
```

Comment:
Can I suggest that we build this functionality into the `Input` component, so that you can use it like:

```js
<Input tooltip={intl.formatMessage({ ...messages.memo_help })}/>
```

Note that the `Input` element already supports a `label` property which can be used to add the `<label>` element automatically. I'm not sure why that isn't being used for this input but we should probably update this so that it does.

Reply:
I will do that.

In `Request.js` the memo field is actually a textarea hence why label component is used instead of the label prop. `<Input />` component can't render a textarea, right?

Reply:
The Pay/Request forms we built before the `label` property was added to the `Input` and `TextArea` fields and these forms had not yet been updated to use the new `label` property.

We have now updated the code in `next` so that these these forms both now properly use the `label` property - which is preferable as it ensures consistent styling of the labels elements.

So - we will need to add the tooltip support directly to these Input/TextArea elements and use via those, so that they can be applied along with the label in a standard and consistent way throughout the app's forms.